src/components/kpis-bar/kpis-bar.tmpl.html

```diff
@@ -2,23 +2,29 @@
 
   <div class="actions">
     <button type="button" class="toggle-show-content im-fab im-primary" ng-click="toggleShowContent()">
-      <a href=""><i class="fa" ng-class="{ 'fa-chevron-up': showContent, 'fa-chevron-down': !showContent }"></i></a>
+      <a href="">
+        <i class="fa" ng-class="{ 'fa-chevron-up': showContent, 'fa-chevron-down': !showContent }" />
```

Comment:
You could use the ternary operator:
`<i ng-class="showContent ? 'fa-chevron-up' : 'fa-chevron-down'"></i>`